src/app/inventory/item-move-service.ts

```diff
@@ -64,12 +65,27 @@ export interface MoveSession {
   bucketsFullOnCurrentStore: Set<number>;
   /** A token that can be checked to see if the whole operation is canceled. */
   readonly cancelToken: CancelToken;
+  /**
+   * Items explicitly involved in the requested move.
+   * Used to distinguish user-intentional moves vs make-space moves.
+   * Contains instanceIds, or for uninstanced items, item hashes.
+   */
+  involvedItems: Set<string | number>;
```

Comment:
main changes are to this

Reply:
In the past I've been using the `excludes` list for this purpose - it's basically meant for exactly what you documented, but it allows for also listing other items that you want to not get moved around.

Reply:
hmm, so `MoveContenxt.excludes` is evaluated in the context of
- moving items aside
- specific items

but i specifically need to exclude
- a bucket not to move things into
- a lack of an item. a "blank space"

i traced it through and gave it a bit of a try but i don't think it can serve this purpose?